tests/module-for-model-test.js

```diff
@@ -73,17 +74,17 @@ test('WhazzitAdapter is registered for model', function(assert) {
   var model = this.subject(),
     store = this.store();
 
-  assert.ok(store.adapterFor(model.constructor) instanceof WhazzitAdapter);
+  assert.ok(store.adapterFor(model.constructor.modelName) instanceof WhazzitAdapter);
 });
 
-test('WhazzitAdapter is used for `find`', function(assert) {
+QUnit.skip('WhazzitAdapter is used for `findAll`', function(assert) {
```

Comment:
should probably include that reason in a code comment so that others don't have to ask the same question 😉